decidim-core/app/cells/decidim/author/profile_minicard.erb

```diff
@@ -0,0 +1,25 @@
+<div class="p-xs flex--sbc">
+  <div class="mr-s">
+    <div class="card__header collapse">
+      <%= cell("decidim/user_profile", raw_model).user_data %>
+    </div>
+    <div class="card__text card--picture-offset">
+      <%= link_to t("decidim.profiles.show.view_full_profile"), profile_path %>
+    </div>
+  </div>
+  <div class="ml-s">
+    <%= cell "decidim/follow_button", raw_model %>
+    <div>
+      <%= link_to decidim.profile_followers_path(raw_model.nickname), class: "card__link" do %>
+        <strong><%= model.followers_count %></strong> <%= t("decidim.profiles.show.followers") %>
```

Comment:
Don't we need a migration for the counters and something to update them?

Reply:
I couldn't find a way to properly add counters and ensure they have the right number. We are only counting the users the user is following, and ignoring anything else, and I cannot add a counter cache with conditions. Also, since the `followable` relation of `Decidim::Follow` is polymorphic, I can't seem to find a way to add a counter_cache on only some classes... 😕

Reply:
You'll have to do it manually. The code for counter caches is inconsistent and won't even take into account association conditions (`where` and such). I've been there in the past and I ended up adding the count logic on the `commands`, which makes sense and also gives you more control on when to update them.

Reply:
Of course, you'll have to duplicate that logic in the migration, which kind of also makes sense.

Reply:
I've finally added the logic as callbacks in the `Decidim::Follow` model, to ensure the logic is performed every time a follow is created/deleted.